packages/neuron-wallet/tests/services/wallets.test.ts

```diff
@@ -155,6 +156,29 @@ describe('wallet service', () => {
         addressType: AddressType.Receiving,
       },
     }
+
+    wallet5 = {
+      name: 'wallet-test5',
+      id: '',
+      extendedKey: 'a',
```

Comment:
```suggestion
      extendedKey: wallet1.extendedKey,
```

to explicitly state that this is a duplicated wallet